xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd

```diff
@@ -379,6 +382,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Vehicel Modes to which ACCESS RIGHTs  apply.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="TransportMode" type="AllVehicleModesOfTransportEnumeration" minOccurs="0">
```

Comment:
I am a bit unsure about this one: What is the difference between the TransportMode and the previous VehicleModes?
And what would the single TransportMode/TransportSubmode pair express in the context of the OrganisationValidityParametersGroup?

Reply:
@Aurige  I can't add a comment that this is for TPEG compatibility. Either you add it or we do it and afterwards think about it.